src/pages/feedbacks/FeedbackList.vue

```diff
@@ -14,11 +14,11 @@
             :text="optionDetail"
           />
         </BaseSelect>
-        <BaseButton link to="/add">+ Add Feedback</BaseButton>
+        <BaseButton link to="/add" text="+ Add Feedback" />
```

Comment:
can you move it into new lines?

Reply:
I can not, always when I change, after saving it comes back to the one line